cli/utils.py

```diff
@@ -223,6 +224,77 @@ def print_batch_exception(batch_exception):
     log.error("-------------------------------------------")
 
 
+'''
+    Job submission
+'''
+
+def print_jobs(jobs: List[aztk.spark.models.Job]):
+    print_format = '{:<34}| {:<10}| {:<20}'
+    print_format_underline = '{:-<34}|{:-<11}|{:-<21}'
+
+    log.info(print_format.format('Job', 'State', 'Creation Time'))
+    log.info(print_format_underline.format('', '', '', ''))
+    for job in jobs:
+
+        log.info(
+            print_format.format(
+                job.id,
+                job.state,
+                utc_to_local(job.creation_time)
+            )
+        )
+
+
+def print_job(job: aztk.spark.models.Job):
+    print_format = '{:<36}| {:<15}'
+
+    log.info("")
+    log.info("Job             %s", job.id)
+    log.info("------------------------------------------")
+    log.info(print_format.format("State:", job.state))
+    log.info(print_format.format("Transition Time:", utc_to_local(job.state_transition_time)))
+    log.info("")
+
+    print_format = '{:<36}| {:<15}| {:<14}'
+    print_format_underline = '{:-<36}|{:-<16}|{:-<17}'
+    log.info(print_format.format("Applications", "State", "Transition Time"))
+    log.info(print_format_underline.format('', '', '', ''))
+
+    for application in job.applications:
```

Comment:
Before printing applications, I think we should call the print_cluster method here (or some summary of it). It's incredibly valuable to see it inline at this point, especially when the cluster is being provisioned and the applications appear to be waiting or stuck.